Dockerfile

```diff
@@ -1,5 +1,5 @@
 ARG STELLAR_CORE_VERSION=18.5.0-877.d387c6a71.focal
-ARG HORIZON_VERSION=2.15.1-247
+ARG HORIZON_VERSION=2.16.0-250
```

Comment:
Can you also update the `Dockerfile.testing` file with the same value?

Reply:
yes, thanks! will note that in the horizon release checklist to include both.